nextcord/client.py

```diff
@@ -1950,18 +1951,33 @@ async def process_application_commands(self, interaction: Interaction) -> None:
                     f"nextcord.Client: Interaction command not found, attempting to lazy load."
                 )
                 # _log.debug(f"nextcord.Client: %s", interaction.data)
-                response_signature = (
+                debug_response_signature = (
```

Comment:
What's the point of this when there's `response_signature`

Reply:
Not sure but `response_signature` is defined below as a dict as is passed to `get_application_command_from_signature` when this is only for the debug log.

The `guild_id` seems to be different 🤷🏼